python/ray/_private/runtime_env/context.py

```diff
@@ -67,20 +71,19 @@ def exec_worker(self, passthrough_args: List[str], language: Language):
             class_path_args = ["-cp", ray_jars + ":" + str(":".join(local_java_jars))]
             passthrough_args = class_path_args + passthrough_args
         elif language == Language.JULIA:
-            executable = "julia"
-            args = [
-                "-e", "'using Ray; start_worker()'",
-                "--"
-            ]
-            # TODO(omus): required to avoid escaping the Julia code. Ideally
-            # this information would be passed in via the serialized runtime
-            # context.
+            executable = self.executable or "julia"
+            args = self.args or ["-e", "'using Ray; start_worker()'"]
```

Comment:
do we ever NOT want to do this?  would it make more sense to allow users to pass _extra_ arguments that are added in addition to these defaults?

Reply:
ah I see, that's `passthrough_args`

Reply:
We can definitely drop the fallback but we probably want to keep it at least temporarily as otherwise we'll break Ray.jl CI when this is merged and https://github.com/beacon-biosignals/Ray.jl/issues/103 has not yet been merged